clippy_lints/src/types.rs

```diff
@@ -1923,12 +1923,21 @@ declare_clippy_lint! {
     /// **Example:**
     /// ```rust
     /// # use std::collections::HashMap;
-    /// # use std::hash::Hash;
+    /// # use std::hash::{Hash, BuildHasher};
     /// # trait Serialize {};
     /// impl<K: Hash + Eq, V> Serialize for HashMap<K, V> { }
     ///
     /// pub fn foo(map: &mut HashMap<i32, i32>) { }
     /// ```
+    /// could be rewritten as
+    /// ```rust
+    /// # use std::collections::HashMap;
+    /// # use std::hash::Hash;
```

Comment:
You'll have to use BuildHasher here as well

Reply:
Wow, the CI even fails on it. Rust's tooling keeps impressing me!